src/deviceCommunication/commDevice.h

```diff
@@ -133,7 +133,7 @@ class CommDevice : public QObject {
      *
      * @param value
      */
-    void newForceDevice(float value);
+    void newForceDevice(float time, float value);
```

Comment:
More than design decision:
Do we want to add the time already in the device class or only in the mainwindow?

Affects:
`commDevice.h`
`commMaster.h` `commMaster.cpp`
`commUSB.h` `commUSB.cpp`

Reply:
I think the comm* classes should only concern themselves with communication. Since the time is already tracked in `Plot` (i.e. through the previous data points) let's just do it there for example with a `Plot::addConsecutiveSample()` method.
When we implement multi-device capability, we can maybe add a `deviceId` field to `Sample` to identify from which device it came. Whether the plot does the translation from `deviceId` to graph itself we can decide then.

Reply:
Agree with the proposal, add a `Plot::addConsecutiveSample()` to plotWidget.h and leave the comm* files as before.

```
void Plot::addConsecutiveSample(Sample reading) {
    addData(lastTime + 1/reading.frequency, reading.measuredValue);
}
```

The call of this function will be implemented on `feature/integrateParser`